### PR TITLE
Separate k8s

### PR DESCRIPTION
**Pull request by CosminBriscaru**

### CODE REVIEW

File: gke.tf [hunk 1]
```diff
@@ -1,30 +1,16 @@
 locals {
```
Comment (mihneastaub) [inline gke.tf:1]: I think it will me much easier, if this module it will increase, to have a separate file named `locals.tf`. But this how you want :D.

**Comment by mihneastaub** [comment]
I think it will be valuable if you can create a documentation for this terraform modules (e.g like this [one](https://github.com/terraform-aws-modules/terraform-aws-eks#requirements)). You can use the tool [terraform-docs](https://github.com/terraform-docs/terraform-docs) is very easy to use.

**Comment by mihneastaub** [comment]
When you have time you can run the terraform fmt on all files.

**Comment by mihneastaub** [comment]
> When you have time you can run the terraform fmt on all files.

I see that this was already done :D

**Comment by mihneastaub** [comment]
How do you test this module?

File: gke.tf (hunk 1 above)
Comment (CosminBriscaru) [reply gke.tf:1]: Might be a preference for me to have this in the files where the local var is used, we'll get back to this later.

**Comment by CosminBriscaru** [comment]
> I think it will be valuable if you can create a documentation for this terraform modules (e.g like this [one](https://github.com/terraform-aws-modules/terraform-aws-eks#requirements)). You can use the tool [terraform-docs](https://github.com/terraform-docs/terraform-docs) is very easy to use.

Will add better docs with the PRs to come as I'm planning to make more changes on these modules.

**Comment by CosminBriscaru** [comment]
> How do you test this module? Maybe in the future we can implement together some automatic testing with [terratest](https://terratest.gruntwork.io/docs/getting-started/quick-start/)🗡️

For now is manually, but I have in mind more automatic things like these so let's discuss them in our calls.

**Review by mihneastaub — APPROVED** [review]
Is look awesome!

**Review by mihneastaub — APPROVED** [review]
Wonderful!

**Pull request merged**